go/internal/flag/flag.go

```diff
@@ -70,6 +70,17 @@ func Parse(fs *flag.FlagSet) {
 	flag.Parse()
 }
 
+// IsFlagPassed returns if the given flag has been provided by the user explicitly or not
```

Comment:
suggestion: IsFlagProvided
WDYT? also @ajm188

Reply:
I don't have strong opinions. Whatever we feel is best. I'll change it to IsFlagProvided for now.